index.html

```diff
@@ -259,10 +259,200 @@ <h1>Architecture</h1>
                 alt="Discovery process overview" />
             <figcaption>Discovery process overview</figcaption>
         </figure>
-        <p class="ednote" title="Discovery Architecture Overview">
-        To do: an overview of the two-phase approach and its purpose, which is to support
-        controlled and authenticated access to metadata by authorized users only.
+	<p>
+	The WoT discovery process is designed using a two-phase approach.
+	In the first phase, one or more of a set of "Introduction" mechanisms may be used to 
+	generate a set of candidate URLs.  These URLs do not themselves contain metadata,
```

Comment:
Is this the process of discovering a single Thing? Why more than one Intro mechanism is used? How do the clients decide between the candidate URLs?

Reply:
WoT Discovery gives back a set of TDs.  The application then needs to decide what to do with them (Consume them all, pick one and Consume it, etc).  We do not define what clients do with the results of Discovery, just the Discovery process itself.